diff --git a/Scene.cpp b/SceneEdited.cpp
index cb7be813b5..ba49cd151d 100644
--- a/Scene.cpp
+++ b/SceneEdited.cpp
@@ -47,6 +47,7 @@ namespace AZ
                 
                 // Set value for constants defined in SceneTimeSrg.azsli
                 scene->m_timeInputIndex = scene->m_srg->FindShaderInputConstantIndex(Name{ "m_time" });
+                scene->m_prevTimeInputIndex= scene->m_srg->FindShaderInputConstantIndex(Name{ "m_prevTime" });
             }
 
             scene->m_name = sceneDescriptor.m_nameId;
@@ -448,6 +449,7 @@ namespace AZ
         {
             AZ_PROFILE_SCOPE(RPI, "Scene: Simulate");
 
+            m_prevSimulationTime = m_simulationTime;
             m_simulationTime = simulationTime;
 
             // If previous simulation job wasn't done, wait for it to finish.
@@ -516,6 +518,7 @@ namespace AZ
             {
                 if (m_timeInputIndex.IsValid())
                 {
+                    m_srg->SetConstant(m_prevTimeInputIndex, m_prevSimulationTime);
                     m_srg->SetConstant(m_timeInputIndex, m_simulationTime);
                 }
 
